### ISSUE TITLE
Block updates not being rendered

### ISSUE THREAD

**Issue by Razaekel** [TallWorlds/CubicChunks#37]
Ref. #34 

"after some respawning, blocks placement in y<0 just deletes the block at wich your pointr is at, instead of placing a block (solved by relogging)"

**Comment by Razaekel** [comment]
Looks like block updates aren't being sent to the client immediately, if the issue is fixed after re-logging. Please confirm that re-logging fixes it.

**Comment by kvverti** [comment]
Yes, I had this in the previous version and I believe that it is a vanilla server bug as well.
